prow/config/jobs/proxy-1.13.yaml

```diff
@@ -504,7 +504,6 @@ jobs:
   - presubmit
 - command:
   - ./prow/proxy-presubmit-centos-release.sh
-  image: gcr.io/istio-testing/build-tools-centos:master-2022-01-19T18-35-09
```

Comment:
this can't just be removed- needs to reference release-1.13, but not removed entirely?

Reply:
The overall reference to 1.13 is on line 3.

Reply:
I see centos now, which is why it wasn't found in searches in my last PR. Update coming.

Reply:
Done.